### PR TITLE
Update to LTS 17

### PR DESCRIPTION
**Pull request by koterpillar**

### CODE REVIEW

**Review by dmoverton — APPROVED** [review]

**Comment by koterpillar** [comment]
Argh, glibc problems again.

**Comment by koterpillar** [comment]
Also `--docker` seems to be on its last legs: https://github.com/commercialhaskell/stack/issues/5489

**Comment by koterpillar** [comment]
Completely crazy option: https://stackoverflow.com/questions/847179/multiple-glibc-libraries-on-a-single-host/44710599#44710599

File: serverless-haskell.hsfiles [hunk 1]
```diff
@@ -190,7 +190,7 @@ plugins:
 #
 # resolver: ./custom-snapshot.yaml
 # resolver: https://example.com/snapshots/2018-01-01.yaml
-resolver: lts-16.3
+resolver: lts-17.4
```
Comment (dmoverton) [inline serverless-haskell.hsfiles:193]: Why is this 17.4 when `stack.yaml` uses 17.7?

Comment (koterpillar) [reply serverless-haskell.hsfiles]: Fixed!

**Review by dmoverton — APPROVED** [review]

**Comment by danwdart** [comment]
> An involved option that requires additional dependencies, but frees this project from glibc constraint entirely: https://stackoverflow.com/questions/847179/multiple-glibc-libraries-on-a-single-host/44710599#44710599

Seems an interesting way to not have to deal with #157...

**Pull request merged**